recipe/bld.bat

```diff
@@ -18,3 +22,17 @@ if errorlevel 1 exit 1
 
 nmake install
 if errorlevel 1 exit 1
+
+@rem Build static libraries, install as re2_static.lib
+cmake -DBUILD_SHARED_LIBS=OFF ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -G "NMake Makefiles" ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
You are not doing an nmake install here, not sure if that is needed.

Reply:
Can't do that because it would clobber the DDL import lib re2.lib

Reply:
Ahh Ok, that makes sense.  Is there a way we could turn echo on, so that we can see what is actually being run?